db/migrate/20170703094235_add_hash_expression_to_miq_alert.rb

```diff
@@ -0,0 +1,30 @@
+class AddHashExpressionToMiqAlert < ActiveRecord::Migration[5.0]
+  class MiqAlert < ActiveRecord::Base
+  end
+
+  def up
+    add_column :miq_alerts, :hash_expression, :text
```

Comment:
Just wondering if this is the best name, because "expression" makes me think of `MiqExpression`, which are also serialized as a hash. Are they called "hash expressions" on the provider side?

Reply:
Yeah, I wasn't sure about the name either, but since I really couldn't think of anything better, I passed on it.  Do you have any suggestions?

Reply:
I don't, which I know makes this comment not much help ;) But if I understand this correctly (I may not), I thought this "expression" was used in the API call to the provider, so would suggest keeping it uniform with whatever they call it?

Reply:
I couldn't come up with a better name either :\ @imtayadeway I'm not sure I understand what do you mean by the "provider side", we plan to use this field in API calls from an ansible module when creating new policies.

Reply:
@zakiva I probably got some details wrong then :) If it's only meaningful in the context of ansible when creating new policies....possible `policy_options` or `policy_conditions`? @Fryguy ?

Reply:
¯\\\_(ツ)_/¯  I'll let you two duke it out 😆

Reply:
@zakiva if neither of those seem appropriate I'm OK with leaving this as is 👍

Reply:
I think this is something other then a policy_x , this is a related to a `miq_alert` not `miq_policy`?
:+1: for leaving as is

Reply:
Thanks, keeping it as is.